config/nrfconnect/chip-module/Kconfig.features

```diff
@@ -35,9 +35,7 @@ endif # CHIP_QSPI_NOR
 
 config CHIP_SPI_NOR
 	bool "Enable SPI NOR feature set"
-	imply SPI
-	imply SPI_NOR
-	imply MULTITHREADING
+	default y if SPI_NOR
```

Comment:
Ditto, I don't think we need this symbol if it's going to be set by nrf symbol.

Reply:
Btw who is going to set `SPI_NOR` now? I don't see it in Matter samples

Reply:
Driver itself according to DTS

Reply:
Cool, it must be something "new", because few years back it was not working like this.